dateparser/timezone_parser.py

```diff
@@ -44,6 +44,16 @@ def pop_tz_offset_from_string(date_string, as_offset=True):
         return date_string, None
 
 
+def is_word_match_any_tz(word):
+    # The regex only interests in timezone not at the beginning
+    word = ' ' + word
+    for name, info in _tz_offsets:
+        timezone_re = info['regex']
+        if timezone_re.match(word):
```

Comment:
Could we work on eliminating false positives a bit more? The reason is that there's quite a lot of time zone abbreviations that exist as words in some languages, .e.g.: WITA, VET, UT, SWAT, NUT.
I think that would require accepting only timezones in all caps, and perhaps also modifying the timezone regex with additional conditions, to avoid matching a timezone in regular sentence, e.g.:
`(?:<?(AM|PM|\d|\s?))timezone_regex`.
Not sure about this though, maybe it would be easier to only match timezones next to already detected time components of a date? This couldn't be done on single token level though.

Reply:
I fixed it to check case-sensitively, which means only in all caps matches it.
Adding additional conditions here needs more change, as it tests the tokenized word only.
But we have something similar in the caller of this function, it only runs this test if any previous token matches a digit, month, week, etc. It only helps to improve performance as it does not need to test the regex for every word.